components/Home/Header/index.tsx

```diff
@@ -0,0 +1,18 @@
+import styles from '../../../styles/Home.module.css'
```

Comment:
You can define in tsconfig paths https://stackoverflow.com/questions/43281741/how-to-use-paths-in-tsconfig-json, so you could have imports done relative to module like ``styles/Home.module.css``